docs/automated_evaluation/automated_evaluation_overview.md

```diff
@@ -41,39 +52,52 @@ competition:
 
 1. To run the automated evaluation you must have docker installed. The instructions for installing Docker Desktop (a GUI program that interfaces with Docker) are found [here](https://docs.docker.com/desktop/install/ubuntu/). The instructions to install Docker Engine with the commanand-line interface only are found [here](https://docs.docker.com/engine/install/ubuntu/). Either should work for testing the automated evaluation.
 
-2. After docker is installed pull the ARIAC image from docker hub using the terminal command:
+2. In order to enable Nvidia GPU acceleration for the docker container you must install the [Nvidia Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker). 
+
+3. After docker is installed pull the ARIAC image from docker hub using the terminal command:
 
-    `docker image pull nistariac/ariac2023:latest`
+    `docker image pull nistariac/ariac2024:latest`
 
-3. Next ensure that the most up-to-date version of ARIAC2023 (at least version 1.2) is on your machine. You can ensure this by running `git pull` from `~/ariac_ws/src/ariac`
+4. Next ensure that the most up-to-date version of ARIAC2023 (at least version 1.2) is on your machine. You can ensure this by running `git pull` from `~/ariac_ws/src/ariac`
 
-4. Navigate to the `automated_evaluation` folder
+5. Navigate to the `automated_evaluation` folder
 
     `cd ~/ariac_ws/src/ariac/automated_evaluation`
 
-5. Add your team configuration file to this folder. It can be named however you like.
+6. Add your team configuration file to this folder. It can be named however you like.
 
-6. Ensure that the `build_container` and `run_trial` scripts can be run as executables:
+7. Ensure that the `build_docker_image`, `build_container` and `run_trial` scripts can be run as executables:
```

Comment:
competitors shouldn't have to build the image so we can remove that script from the instructions